### DISCUSSION TITLE
Purchase Order customisation

### DISCUSSION THREAD

**Discussion by lokyi792** [inventree/InvenTree#7592] [Q&A]
Preface this by admitting I have no experience with generating HTML code - other than cutting and pasting the example PO code from github into the template PO report... which I can loosely comprehend.
[https://github.com/inventree/InvenTree/blob/0.15.x/src/backend/InvenTree/report/templates/report/inventree_po_report_base.html](url)

One thing I find curious is the use of the supplier image on the purchase order.
Two parts to this question:
1) Where/how can upload our own company logo?
2) How to reference that to the PO?

I also need to work out other basic stuff like shipping addresses etc.

**Comment by SchrodingersGat** [comment]
@lokyi792 you can upload "assets" for use in reports / labels, as documented here:

https://docs.inventree.org/en/stable/report/report/#report-assets

There are also helper functions for embedding and encoding images into your report template:

https://docs.inventree.org/en/stable/report/helpers/#uploaded-images
